src/SumoCoders/FrameworkCoreBundle/Resources/views/base.html.twig

```diff
@@ -23,7 +23,7 @@
     <![endif]-->
 
     {% block stylesheets %}
-      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/3.5.4/select2.min.css" />
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/css/select2.min.css" />
```

Comment:
I had started typing it and then thought, nah, we can leave it like this, but apparently I accidentally submitted it anyway